README.md

```diff
@@ -611,6 +616,82 @@ The view function here is only an example. In practice thunks are only
 relevant if you are rendering a complicated view that takes
 significant computational time to generate.
 
+## JSX
+
+[JSX](https://facebook.github.io/jsx/) is an XML-like syntax extension to JavaScript (ECMAScript).
+Instead of using `h(tag, props, children)` to define the virtual tree,
+Snabbdom supports using JSX with transpilers such as Babel and Typescript.
+
+Built-in `jsx` function is a very thin and performant layer on top of `h`.
+Top level props are the initialized [modules](#modules-documentation) such as `class`, `attrs`, `props`, `on`, `style`.
```

Comment:
What about `hooks`? Are they supported?

Reply:
yeah, it re-routes to underlying h, so everything is supported. I can add `hooks` to the docs.

Reply:
Thanks for clarification. It would be nice to add.